.env.test

```diff
@@ -9,3 +9,7 @@ SECRET_KEY=secret-key
 # Database
 CONNECTION_STRING=mongodb://fakemongo.com
 DATABASE_NAME=fakeDbName
+
+# Authentication
+
+LOGIN_DISABLED=True
```

Comment:
It would be nice to add this (set to false) and the other new id/secret to the `.env.template` file